lib/fittings/error_handler.js

```diff
@@ -23,9 +25,15 @@ module.exports = function create() {
                 res.locals.errorStatus = err.status;
             }
 
-            var stackDump = err.stack.replace(/\n/g, '');
             message = message.replace(/\n/g, '');
-            res.body = {message: message, status: res.locals.errorStatus, stack: stackDump};
+            res.body = {message: message, status: res.locals.errorStatus};
+            var minLogLevel = nconf.get('minLogLevel');
+            if (minLogLevel === constants.Logging.Levels.debug) {
+                if (err.stack !== undefined) {
+                    var stackDump = err.stack.replace(/\n/g, '');
```

Comment:
@dalebremner In that case, could we change the template to expect `stack` to be an array of strings and use a forEach to render them all on separate lines?